### PR TITLE
Inline form demo

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: src/app/demos/inline-form/inline-form-demo.component.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
+<div
+  id="example-simple"
+  class="sky-margin-stacked-separate"
+>
+  <h3>Simple example</h3>
```
Comment (blackbaud-johnly) [inline src/app/demos/inline-form/inline-form-demo.component.html]: I think I'd go with "Inline form with Save and Cancel buttons" for this header.

File: src/app/demos/inline-form/inline-form-demo.component.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    </button>
+  </h4>
+  </sky-inline-form>
+</div>
+
+<div
+  id="example-simple"
+  class="sky-margin-stacked-separate"
+>
+  <h3>Custom buttons example</h3>
```
Comment (blackbaud-johnly) [inline src/app/demos/inline-form/inline-form-demo.component.html]: I think I'd go with "Inline form with custom buttons" for this header.

File: src/app/demos/inline-form/inline-form-demo.component.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  </sky-repeater>
+
+</div>
+
+
+<div
+  class="sky-margin-stacked-separate"
+  id="example-tiles"
+>
+  <h3>Tile sections using inline-form</h3>
```
[the remark this answers is not in the document]
Comment (Blackbaud-AlexKingman) [reply src/app/demos/inline-form/inline-form-demo.component.html]: You can find the tile content in `inline-form-demo-tile.component.ts` and `inline-form-demo-tile.component.html`.

**Comment by Blackbaud-AlexKingman** [comment]
@blackbaud-johnly I think I've seen to all your feedback. Let me know what you think! Thanks.

**Pull request merged**